docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,93 @@
+
+    bash ${CURRENT_DIR}/../scripts/create-cr.sh --output "${CR_PATH}" --domain "${DOMAIN}"
+    bash ${CURRENT_DIR}/../scripts/installer.sh --local --cr "${CR_PATH}"
+
+    rm -rf $TMPDIR
+else
+    bash ${CURRENT_DIR}/../scripts/installer.sh --cr "${CR_PATH}"
+fi
+```
+
+The subsequent paragraphs describe respective sub-scripts triggered during the installation process.
+
+## The minikube.sh script
+
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
+
+Purpose of `installation/scripts/minikube.sh` is to configure and start Minikube. The script also checks if your development environment is cofigured to handle Kyma installation. This includes checking Minikube and Kubectl versions. If Minikube is already initialized, the system prompts you to agree to remove the previous Minikube cluster. The script exits if you don't want to restart your cluster.
+
+Minikube is configured to disable default nginx ingress controller.
+
+>**NOTE:** For the complete list of parameters passed to the `minikube start` command, refer to the `installation/scripts/minikube.sh` script.
+
+Once Minikube is up and running, the script adds local installation entries to `/etc/hosts`.
+
+## The build-kyma-installer.sh script
+
+Installer is application based on a [Kubernetes operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the `installation` Custom Resource. Kyma-Installer is a Docker image that bundles Installer binary with Kyma charts. 
+
+The `installation/scripts/build-kyma-installer.sh` script extracts Kyma-Installer image name from the `installer.yaml` deployment file and uses it to build a Docker image inside Minikube. This image contains local Kyma sources from `resources` folder. 
+
+>**NOTE:** For Kyma-Installer Docker image details, refer to the `kyma-installer/kyma.Dockerfile` file.
+
+## The generate-local-config.sh script
+
+The Azure-Broker sub-component is a part of the `core` deployment that provisions managed services in the Microsoft Azure cloud. To enable Azure-Broker, export the following environment variables:
+ - AZURE_BROKER_SUBSCRIPTION_ID
+ - AZURE_BROKER_TENANT_ID
+ - AZURE_BROKER_CLIENT_ID
+ - AZURE_BROKER_CLIENT_SECRET
+
+>**NOTE:** As the Azure credentials are converted to a Kubernetes Secret, make sure the exported values are base64-encoded.
+
+## The create-cr.sh script
+
+The `installation/scripts/create-cr.sh` script prepares the `installation` Custom Resource from the `installation/resources/installer-cr.yaml.tpl` template. The local installation scenario uses the default `installation` Custom Resource. Kyma-Installer already contains local Kyma resources bundled, thus `url` is ignored by the Installer component. 
```

Comment:
the Kyma-Installer ? (here and there?)

Reply:
i guess done (here, cause idk where there is).